app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/CheckLiabilityDetailsAnswersController.scala

```diff
@@ -32,18 +32,23 @@ class CheckLiabilityDetailsAnswersController @Inject() (
   authenticate: AuthAction,
   journeyAction: JourneyAction,
   mcc: MessagesControllerComponents,
-  page: check_liability_details_answers_page,
-  liabilityLinkHelper: LiabilityLinkHelper
+  startRegistrationController: StartRegistrationController,
+  page: check_liability_details_answers_page
 ) extends FrontendController(mcc) with I18nSupport {
```

Comment:
Could this be a `LiabilityController`?  .....

Reply:
Most certainly. Good spot.